core/src/main/java/io/github/tcdl/msb/api/monitor/AggregatorStats.java

```diff
@@ -0,0 +1,29 @@
+package io.github.tcdl.msb.api.monitor;
+
+import io.github.tcdl.msb.config.ServiceDetails;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+/**
+ * This class represents internal state of {@link io.github.tcdl.msb.api.monitor.ChannelMonitorAggregator} that is passed to registered handlers.
```

Comment:
what we actually gather here? Internal state of some object sounds like a "magic"